### ISSUE TITLE
Update docker workflow to restore latest.dump 

### ISSUE THREAD

**Issue by managedkaos** [EBWiki/EBWiki#3751]
1. Docker workflow needs a method to load `latest.dump` into the DB running in the container.
2. Update PostgreSQL to v12.

**Comment by managedkaos** [comment]
```
apt-get install lsb-release
wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
echo "deb http://apt.postgresql.org/pub/repos/apt/ `lsb_release -cs`-pgdg main" | tee  /etc/apt/sources.list.d/pgdg.list
apt-get update && apt-get -y install postgresql-12 postgresql-client-12
```

**Comment by managedkaos** [comment]
https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions

for github.sha to use in docker tag: `${{ github.sha }}`

**Comment by managedkaos** [comment]
with PG 12 installed, this works:
```
docker exec -e PGPASSWORD=ebwiki -it ebwiki pg_restore --verbose --clean --no-acl --no-owner -p 5432 -h localhost -U blackops -d blackops_development /usr/src/ebwiki/latest.dump
```

**Comment by managedkaos** [comment]
```
FROM ruby:2.6.5

COPY Gemfile Gemfile.lock /

RUN gem install bundler && bundle install && gem install fakes3
RUN apt-get update -qq && apt-get install -qq --no-install-recommends lsb-release apt-transport-https && \
    wget -q https://artifacts.elastic.co/GPG-KEY-elasticsearch && \
    wget -q https://www.postgresql.org/media/keys/ACCC4CF8.asc && \
    apt-key add GPG-KEY-elasticsearch && \
    apt-key add ACCC4CF8.asc && \
    echo "deb http://apt.postgresql.org/pub/repos/apt/ `lsb_release -cs`-pgdg main" > /etc/apt/sources.list.d/pgdg.list && \
    echo "deb https://artifacts.elastic.co/packages/6.x/apt stable main" > /etc/apt/sources.list.d/elastic-6.x.list && \
    apt-get update -qq && \
    apt-get install -qq --no-install-recommends \
        apt-utils \
        build-essential \
        libpq-dev \
        nodejs \
        npm \
        default-jre \
        postgresql-12  \
        postgresql-client-12  \
        redis-server && \
    apt-get install -qq --no-install-recommends elasticsearch && \
    mkdir /usr/src/ebwiki

WORKDIR /usr/src/ebwiki
EXPOSE 3000
ENTRYPOINT ./dev_provisions/entrypoint.sh

```

**Comment by managedkaos** [comment]
Makefile:
```
build:
	docker build --tag ebwiki/ebwiki .

run:
	docker run --detach --rm --volume $(PWD):/usr/src/ebwiki \
		--publish 3000:3000 --name ebwiki ebwiki/ebwiki
	./dev_provisions/prewarm.sh

updatedb:
	docker exec -e PGPASSWORD=ebwiki -it ebwiki pg_restore --verbose --clean \
		--no-acl --no-owner -p 5432 -h localhost -U blackops \
		-d blackops_development /usr/src/ebwiki/latest.dump

test:
	./dev_provisions/run_tests.sh

logs:
	docker logs --timestamps --follow ebwiki

exec:
	docker exec -it ebwiki bash

stop:
	docker stop ebwiki || echo "stopped"

clean: stop
	docker image rm ebwiki/ebwiki:latest || echo "clean"

all: clean build run

.PHONY: run build exec stop clean all
```

**Comment by managedkaos** [comment]
This issue is closed by #3781.
